indico/modules/events/templates/management/_affiliation.html

```diff
@@ -0,0 +1,19 @@
+{%- macro render_affiliation(person) -%}
+    {% set details = person.affiliation_details %}
+    {%- if details -%}
+        {%- set details = person.affiliation_details -%}
+        {%- set uuid = uuid() -%}
+        <span id="affiliation-popup-container-{{ uuid }}">{# -#}
+            <span id="affiliation-popup-{{ uuid }}"></span>{# -#}
+            <span>{{details.name}}</span>{# -#}
+        </span>{# -#}
+        <script>
+            setupAffiliationPopup(
+                {{ uuid|tojson }},
+                {{ person.affiliation_details|tojson }}
+            );
+        </script>{# -#}
```

Comment:
the `{# -#}` here probably isn't needed; you already strip the whitespace in the line below (`{%- elif`)